pkg/kubectl/cmd/util/helpers.go

```diff
@@ -159,7 +160,13 @@ func checkErr(pref string, err error, handleErr func(string)) {
 
 func statusCausesToAggrError(scs []unversioned.StatusCause) utilerrors.Aggregate {
 	errs := make([]error, len(scs))
```

Comment:
to make sure we don't end up with nil elements in here, allocate with capacity equal to len(scs) but length 0, and use errs = append(errs, ...) rather than assigning to direct indexes in the loop